blockchain/chain_sync/src/sync.rs

```diff
@@ -839,8 +869,9 @@ where
             }
 
             // TODO tweak request window when socket frame is tested
-            const REQUEST_WINDOW: i64 = 5;
+            const REQUEST_WINDOW: i64 = 30;
             let epoch_diff = cur_ts.epoch() - to_epoch;
+            info!("BlockSync from: {} to {}", cur_ts.epoch(), to_epoch);
```

Comment:
can you change this from an info to debug? Don't think this should be logged by default